.github/workflows/node.js.yml

```diff
@@ -28,3 +28,20 @@ jobs:
     - run: npm install
     - run: npm run build --if-present
     - run: npm test
+
+  build-and-deploy:
```

Comment:
We're adding another `job` right, that builds the library to deploy for GitHub pages.

I'd clarify this in the commit message.

Reply:
thanks, adding more info on the PR description.